api/pkg/app/gorm_logger.go

```diff
@@ -55,7 +55,8 @@ func (w *prodWriter) Printf(format string, data ...interface{}) {
 	msg := ""
 
 	for i, d := range data {
-		switch d.(type) {
+		dt := d
```

Comment:
can't we remove this `dt` since we are using this only in the switch condition?

Reply:
Added `dt` to resolve the lint errors